src/imXpadCamera.cpp

```diff
@@ -162,10 +163,6 @@ int Camera::init()
 
 	DEB_TRACE() << "********** Outside of Camera::init ***********";
 	
-	//use module mask to enable/disable some modules. 
-	//Do not apply if 0, in order to keep compatibility with others versions
-	if(m_module_mask!=0)
-		setModuleMask(m_module_mask);
```

Comment:
this function is now never called ...
check with another detector that has several modules